DataConnectors/WireXsystemsNFP.json

```diff
@@ -0,0 +1,119 @@
+{

```

Comment:
This file is the same as the first data connector file. Please remove the dupe files from this PR. Just one data connector file is needed unless there are two different types of data connectors included for WireX.

Reply:
dropped new edited file